### ISSUE TITLE
Integrations with cardbook no longer working

### ISSUE THREAD

**Issue by adiestone** [thundernest/quicktext#383]
Yesterday all of my templates in Thunderbird stopped working.

They no longer insert any of the TO variables into the body of the email when I select a template.

I don't know if this is a quicktext, thunderbird or cardbook issue so have opened this on all platforms.

Has anyone else experienced issues?

Quicktext v5.14
Cardbook v91.6
Thunderbird v115.4.1

**Comment by herbie4** [comment]
Yes, here it is not working any more too. Quicktext is using address book instead of cardbook entries. 
tb version: 140.7
qt version: 6.5.5
cb version: 104.8
